client/driver/docker.go

```diff
@@ -91,6 +91,10 @@ const (
 
 type DockerDriver struct {
 	DriverContext
+
+	imageID      string
+	waitClient   *docker.Client
```

Comment:
You don't have to pass this as it is a shared variable that is initialized once per client